src/widgets/MultiLineTextInput.js

```diff
@@ -76,8 +76,6 @@ export default class MultiLineTextInput extends createjs.Container {
     this._height = height;
 
     this.addEventListener('click', this.onFocus);
-    // todo: handle mouse based blur causing event
-
```

Comment:
Nit: technically the todo's in this file should remain since fixing text input by moving focus to the DOM translation will be a separate PR.

Reply:
reverted